src/components/footer/Footer.jsx

```diff
@@ -1,13 +1,16 @@
 const footerList = [
```

Comment:
consider putting static data in different file and use Footer component to just render the data.

Reply:
done